src/evm/onchain/endpoints.rs

```diff
@@ -66,7 +75,7 @@ impl Chain {
     pub fn get_chain_rpc(&self) -> String {
         match self {
             Chain::ETH => "https://eth.llamarpc.com",
-            Chain::BSC => "https://bsc.llamarpc.com",
+            Chain::BSC => "https://quiet-omniscient-daylight.bsc.quiknode.pro/88e8c7767457e3c8a2dc8ba889e18f63bd2a5b5d/",
```

Comment:
dont push our private endpoint, please undo last commit and use force push to overwrite it